### PR TITLE
Use `wp_cache_get_multiple` in `fill_descendants`.

### PR DESCRIPTION
**Pull request by spacedmonkey**
Trac ticket:  https://core.trac.wordpress.org/ticket/57803

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by tillkruss** [comment]
LGTM 👍

File: src/wp-includes/class-wp-comment-query.php [hunk 1]
```diff
@@ -1036,13 +1036,19 @@ protected function fill_descendants( $comments ) {
...
 			$uncached_parent_ids = array();
 			$_parent_ids         = $levels[ $level ];
-			foreach ( $_parent_ids as $parent_id ) {
-				$cache_key        = "get_comment_child_ids:$parent_id:$key:$last_changed";
-				$parent_child_ids = wp_cache_get( $cache_key, 'comment' );
-				if ( false !== $parent_child_ids ) {
-					$child_ids = array_merge( $child_ids, $parent_child_ids );
-				} else {
-					$uncached_parent_ids[] = $parent_id;
+			if ( $_parent_ids ) {
```
Comment (mukeshpanchal27) [inline src/wp-includes/class-wp-comment-query.php:1033]: Thanks for checking, as it will not load code for posts that do not have comments.

**Review by mukeshpanchal27 — APPROVED** [review]
Thanks @spacedmonkey, The changes look good to me.

**Comment by spacedmonkey** [comment]
Committed.

**Pull request closed without merging**